packages/manager/modules/pci/src/projects/project/users/roles-matrix/roles-matrix.less

```diff
@@ -26,5 +26,10 @@
       border-left: none;
       box-shadow: none;
     }
+
+    .oui-datagrid__header {
```

Comment:
Maybe this should be done in `ui-kit`, it looks like the best behaviour for most datagrids

Reply:
No, Not in this form. 
It's too specific for this case since the datagrid is designed to have a responsive form with the scrollbar. For example, this will have collateral effect with the sticky cells, that will not adapt their heights with the non-sticky cells.